apps/aestratum/src/aestratum_env.erl

```diff
@@ -0,0 +1,48 @@
+-module(aestratum_env).
+
+%% ENV handling
+-export([setup/0,
+         get/1,
+         get/2,
+         set/1,
+         set/2,
+         reset/1,
+         unset/0, unset/1,
+         keys/0]).
+
+%%%%%%%%%%
+
+setup() ->
+    unset(),
```

Comment:
Why? If correct, why not doing it for all apps?

If only for tests, then tests shall rather be amended avoiding leaking configs.

Reply:
Not sure which bit is in question but this all this is stratum specific, due to its more complex startup - possibly creating mnesia tables at different time the rest of node does...